Source/Widgets/InfoBox/InfoBox.js

```diff
@@ -200,5 +234,22 @@ click: function () { closeClicked.raiseEvent(this); }');
         return destroyObject(this);
     };
 
+    InfoBox.stringToHtml = function stringToHtml(s, frame) {
+        // if we don't have a script tag - in case the method was used as static...
+        if (s.indexOf('<script') > -1) {
```

Comment:
I think we should check for both an opening and closing script tag for security purposes before adding it to the document.

Reply:
There is definitely more to do in order to be thorough about security. I think since allowing scripts is off by default, and since the docs warn about the risks of turning it on, we don't have to add sanitization here and can leave that up to the user instead.